src/main/java/edu/ucsb/cs156/happiercows/repositories/CommonsRepository.java

```diff
@@ -8,5 +10,5 @@
 
 @Repository
 public interface CommonsRepository extends CrudRepository<Commons, Long> {
-    
+    Optional<Commons> findById(Long id);
```

Comment:
Is this necessary? I believe `findById` is already provided by `CrudRepository`, so this definition would be redundant.

Reply:
Apologies. Will immediately fix.